.github/workflows/skywalking.yaml

```
@@ -641,6 +641,8 @@ jobs:
             config: test/e2e-v2/cases/kafka/kafka-monitoring/e2e.yaml
           - name: MQE Service
             config: test/e2e-v2/cases/mqe/e2e.yaml
+          - name: Pulsar
+            config: test/e2e-v2/cases/pulsar/e2e.yaml
```

```suggestion
          - name: Pulsar and BookKeeper
            config: test/e2e-v2/cases/pulsar/e2e.yaml
```

We are expecting you have tests/validation for metrics of Pulsar and BookKeeper.